src/Microsoft.Health.Dicom.Functions/Indexing/Models/ReindexInstanceActivityInput.cs

```diff
@@ -0,0 +1,31 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using Microsoft.Health.Dicom.Core.Features.ExtendedQueryTag;
+
+namespace Microsoft.Health.Dicom.Functions.Indexing.Models
+{
+    /// <summary>
+    ///  Represents input to <see cref="ReindexDurableFunction.ReindexInstanceActivityAsync"/>
+    /// </summary>
+    public class ReindexInstanceActivityInput
+    {
+        /// <summary>
+        /// Gets or sets the inclusive start watermark.
+        /// </summary>
+        public long StartWatermark { get; set; }
+
+        /// <summary>
+        /// Gets or sets the inclusive end watermark.
+        /// </summary>
+        public long EndWatermark { get; set; }
```

Comment:
If its easier to user, we can also use a tuple here

Reply:
To use elements within tuple, need to call like `Tuple.Item1` `Tuple.Item2`, which I think unclear.